### PR TITLE
Don't panic in continue_run if no groups found within range  

### PR DESCRIPTION
**Pull request by Azrenbeth**
Also allow `groups_to_compress` to be used if `min_state_group` is None
(previously had to set it to something like -1 which was ugly)

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -548,48 +553,28 @@ pub fn continue_run(
...
-        compressor.stats.resets_no_suitable_prev
-    );
-    println!(
-        "  Number of compressed rows caused by the above: {}",
-        compressor.stats.resets_no_suitable_prev_size
-    );
-    println!(
-        "  Number of state groups changed: {}",
-        compressor.stats.state_groups_changed
-    );
```
Comment (erikjohnston) [inline src/lib.rs:573]: Why are we removing these?

Comment (Azrenbeth) [reply src/lib.rs:573]: I didn't actually mean to add them earlier! The idea is that all this already being given in the `ChunkStats` struct (what `continue_run` returns) and so the caller can do the output they want.

Comment (erikjohnston) [reply src/lib.rs:573]: Aha! Didn't realise these had been added

**Review by erikjohnston — APPROVED** [review]

File: src/lib.rs (hunk 1 above)
Comment (Azrenbeth) [reply src/lib.rs:573]: They're still in `run()` (what the manual command line tool triggers) but I remove them here from `continue_run()`

**Pull request merged**